boa/src/builtins/bigint/mod.rs

```diff
@@ -0,0 +1,99 @@
+//! This module implements the global `BigInt` object.
+//!
+//! `BigInt` is a built-in object that provides a way to represent whole numbers larger
+//! than the largest number JavaScript can reliably represent with the Number primitive
+//! and represented by the `Number.MAX_SAFE_INTEGER` constant.
+//! `BigInt` can be used for arbitrarily large integers.
+//!
+//! More information:
+//!  - [ECMAScript reference][spec]
+//!  - [MDN documentation][mdn]
+//!
+//! [spec]: https://tc39.es/ecma262/#sec-bigint-objects
+//! [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
+
+use crate::{
+    builtins::{
+        function::make_constructor_fn,
+        value::{ResultValue, Value},
+    },
+    exec::Interpreter,
+    syntax::ast::bigint::BigInt,
+};
+
+#[cfg(test)]
+mod tests;
+
+/// `BigInt()` function.
+///
+/// More Information https://tc39.es/ecma262/#sec-number-constructor-number-value
```

Comment:
This link points to the `Number` constructor, but the BigInt constructor is here:
https://tc39.es/ecma262/#sec-bigint-constructor